### PR TITLE
remove the ophan dependency

### PR DESCRIPTION
**Pull request by QuarpT**
## What does this change?

Remove Ophan dependency from frontend project. 

Maintaining dependencies is expensive and not necessary for two string constants.

## What is the value of this and can you measure success?

One step on the way to the Scala 2.12 upgrade

## Tested in CODE?

Not necessary

### CODE REVIEW

**Review by TBonnin — APPROVED** [review]
looks good to me. Let's double check with @joelochlann so he knows about it and can confirm the strings are correct

File: common/app/navigation/helpers/UrlHelpers.scala [hunk 1]
```diff
@@ -90,10 +88,10 @@ object UrlHelpers {
     }
 
     val acquisitionData = Json.obj(
-      "source" -> AcquisitionSource.GuardianWeb.originalName,
+      "source" -> "GUARDIAN_WEB",
```
Comment (joelochlann) [inline common/app/navigation/helpers/UrlHelpers.scala]: perhaps we could have a comment explaining that this has to correspond to a Thrift enum and linking to https://dashboard.ophan.co.uk/docs/thrift/acquisition.html#Enum_AcquisitionSource?

File: common/app/navigation/helpers/UrlHelpers.scala [hunk 2]
```diff
@@ -90,10 +88,10 @@ object UrlHelpers {
...
       "componentType" -> (position match {
-        case NewHeader | OldHeader | AmpHeader | SideMenu | SlimHeaderDropdown => ComponentType.AcquisitionsHeader.originalName
-        case Footer => ComponentType.AcquisitionsFooter.originalName
+        case NewHeader | OldHeader | AmpHeader | SideMenu | SlimHeaderDropdown => "ACQUISITIONS_HEADER"
+        case Footer => "ACQUISITIONS_FOOTER"
```
Comment (joelochlann) [inline common/app/navigation/helpers/UrlHelpers.scala]: and here we're referencing https://dashboard.ophan.co.uk/docs/thrift/componentevent.html#Enum_ComponentType

**Review by joelochlann — APPROVED** [review]
Looks good, strings are correct - some comments might be useful to prevent mistakes in future now that the compiler won't

File: common/app/navigation/helpers/UrlHelpers.scala (hunk 1 above)
Comment (QuarpT) [reply common/app/navigation/helpers/UrlHelpers.scala]: Sure, I'll add the comments

**Pull request merged**